lib/logging.js

```diff
@@ -17,9 +17,14 @@ function addRequest (nRequest, nResponse, callback) {
   let d = new Date()
   let nMetaInfo = { timestamp: d.valueOf(), serverVersion: packageInfoServer.version, engineVersion: packageInfoEngine.version }
   let newRequest = { request: nRequest, response: nResponse, metaInfo: nMetaInfo }
-
-  Request.create(newRequest)
-  callback(newRequest._id)
+  Request.create(newRequest, (err, savedObject) => {
+    if (err) {
+      // throw new Error('Database error: Error while saving Request')
+      console.log('error in database save: ' + err)
```

Comment:
Das war ja die Idee, ich gebe einfach was in die Konsole aus und "ignoriere" den Error sonst haha. ist ja nichts weltbewegendes wenn das logging ausfällt.